backend/main/src/Group/Controllers/UpdateGroupNameController.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller } from "src/Shared";
+import { UpdateGroupNameUseCase } from "src/Group/UseCases/UpdateGroupNameUseCase/UpdateGroupNameUseCase";
+import {
+	IUpdateGroupNameUseCase,
+	UserIsNotAuthorized,
+} from "src/Group/UseCases/UpdateGroupNameUseCase/IUpdateGroupNameUseCase";
+
+export class UpdateGroupNameController extends Controller<
+	IUpdateGroupNameUseCase,
+	{ groupName: string }
+> {
+	constructor(private readonly updateGroupNameUseCase: UpdateGroupNameUseCase) {
+		super();
+	}
+
+	public async handler(request: IUpdateGroupNameUseCase) {
+		const result = await this.updateGroupNameUseCase.execute(request);
+		if (result.err) {
+			if (result.error instanceof UserIsNotAuthorized) {
+				return this.forbidden(result.error.message);
+			}
+			return this.badRequest(result.error);
+		}
+		return this.ok({ groupName: request.name });
```

Comment:
Based on API doc, it should be 
```suggestion
		return this.accepted()
```

Reply:
Thank you for pointing it out. It's my mistake. And I've fixed it.